product_state/i18n/fr.po

```diff
@@ -0,0 +1,51 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# * product_state
+#
+# Translators:
+# OCA Transbot <transbot@odoo-community.org>, 2017
+# guillaume bauer <guillaume.bauer@syentys.com>, 2017
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-07-29 02:55+0000\n"
+"PO-Revision-Date: 2017-07-29 02:55+0000\n"
+"Last-Translator: guillaume bauer <guillaume.bauer@syentys.com>, 2017\n"
+"Language-Team: French (https://www.transifex.com/oca/teams/23907/fr/)\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. module: product_state
+#: selection:product.template,state:0
+msgid "End of Lifecycle"
+msgstr "Fin de vie"
+
+#. module: product_state
+#: selection:product.template,state:0
+msgid "In Development"
+msgstr "Phase de recherche et de développement"
```

Comment:
This french translation is far too long (this is not a help, but the field title). Further more, it litteraly means "Research and development phase" while the original text is different. I sggest to write "En développement".